client/src/api/translations.js

```diff
@@ -13,6 +13,21 @@ export const pickDocument = id => {
   return fetch("/api/translations", userRequest);
 };
 
+export const cancelDocument = id => {
```

Comment:
The responsibility of this function is to cancel a translation so the name should be cancelTranslation and not cancelDocument